src/main/kotlin/solve/styles/Style.kt

```diff
@@ -0,0 +1,24 @@
+package solve.styles
+
+object Style {
+
+
+        const val backgroundColour = "EFF0F0"
+
+        const val surfaceColor = "FFFFFF"
+
+        const val primaryColor = "78909C"
+
+        const val onBackgroundColor = "000000"
+
+        const val primaryColorLight = "B0BEC5"
+
+        const val fontCondensed = "Roboto Condensed"
+
+        const val font = "Roboto"
+
+        const val buttonStyle = "-fx-font-family: Roboto Condensed; -fx-font-size: 14px; -fx-font-weight: Bold; -fx-text-fill: #78909C;"
+
+        const val fontSize = "14px"
```

Comment:
Specify where this font size should be used, use it everywhere it is needed, and do it for other sizes (20, 15, etc)